spec/index.bs

```diff
@@ -7367,19 +7381,11 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
     Return `true` if all of the following conditions are met, and `false` otherwise:
 
     - |descriptor|.{{GPUProgrammableStage/module}} must be a [=valid=] {{GPUShaderModule}}.
-    - If |descriptor|.{{GPUProgrammableStage/entryPoint}} is [=map/exists|provided=]:
-
-        - |descriptor|.{{GPUProgrammableStage/module}} must contain exactly one
-            entry point with a name equalling |descriptor|.{{GPUProgrammableStage/entryPoint}},
-            and its shader stage must equal |stage|.
-
-        Otherwise:
-
-        - |descriptor|.{{GPUProgrammableStage/module}} must contain exactly one
-            entry point for shader stage |stage|.
-    - For each |binding| that is [=statically used=] by |descriptor|:
+    - Let |entryPoint| be [$get the entry point$](|stage|, |descriptor|).
```

Comment:
nit: It's no longer a list of conditions ("the following conditions...") if one of the lines is just "Let...".

We run into this problem a lot - we use a lot of unordered lists because the spec doesn't prescribe validation order. But it creates a lot of tricky situations like this, where several steps need to run in order before an unordered list. For this case we could either do:

```bikeshed
    Return `true` if all of the following conditions are met, and `false` otherwise:

    - |descriptor|.{{GPUProgrammableStage/module}} must be a [=valid=] {{GPUShaderModule}}.
    - |entryPoint| must not be `null`.
    ...

    where |entryPoint| is [$get the entry point$](|stage|, |descriptor|).
```

or use an ordered list of steps - we are already using this in some of the substeps, so it would be more consistent. We haven't used this header wording before, but it's more explicit what is happening in the list.

```
    Return `true` if all requirements in the following steps are met, and `false` otherwise:

    1. |descriptor|.{{GPUProgrammableStage/module}} must be a [=valid=] {{GPUShaderModule}}.
    1. Let |entryPoint| be [$get the entry point$](|stage|, |descriptor|).
    1. |entryPoint| must not be `null`.
    ...
```

Reply:
The latter seems better to me.